locale/fr.json

```diff
@@ -27,8 +27,8 @@
         "spawning_discord": "**%{servername}** est en train de démarrer."
     },
     "discord": {
-        "status_online": "**%{servername}** est actuellement **Ouvert**!",
-        "status_offline": "**%{servername}** est actuellement **Fermer**!"
+        "status_online": "**%{servername}** est actuellement **Ouvert** !",
+        "status_offline": "**%{servername}** est actuellement **Fermer** !"
```

Comment:
Why this added space?
Someone on discord just told me:
> its not "est actuellement Fermer"
> its "est actuellement Fermé"

Reply:
Yes is "Fermé" not "Fermer".

Reply:
I corrected that thanks
I added the space because in French we have space before an exclamation mark and a question mark